sdk/runanywhere-python/tests/test_client.py

```
@@ -0,0 +1,252 @@
+        self._record("initialize", secure_dir, base_dir)
+
+    def shutdown(self) -> None:
+        self._record("shutdown")
+
+    # -- loads (return a fake handle) --
+    def _handle(self) -> int:
+        h = self._next_handle
+        self._next_handle += 1
+        return h
+
+    def load_model(self, path: str, id: object = None, name: object = None) -> int:
```

Ruff (0.15.21)

[error] Function argument `id` is shadowing a Python builtin

(A002)